.travis.yml

```diff
@@ -18,17 +19,13 @@ matrix:
         - os: osx
           env: QT5=True
 before_install:
-   - . ${TRAVIS_BUILD_DIR}/.travis/${TRAVIS_OS_NAME}.${TARGET_OS}.before_install.sh
+    - . ${TRAVIS_BUILD_DIR}/.travis/before_install.sh
 install:
-    - . ${TRAVIS_BUILD_DIR}/.travis/${TRAVIS_OS_NAME}.${TARGET_OS}.install.sh
+    - . ${TRAVIS_BUILD_DIR}/.travis/install.sh
 before_script:
-    - mkdir build && cd build
-    - export CMAKE_FLAGS="-DWANT_QT5=$QT5 -DUSE_WERROR=ON -DCMAKE_BUILD_TYPE=RelWithDebInfo"
-    - if [ -z "$TRAVIS_TAG" ]; then export CMAKE_FLAGS="$CMAKE_FLAGS -DUSE_CCACHE=ON"; fi
+    - . ${TRAVIS_BUILD_DIR}/.travis/before_script.sh
 script:
-    - . ${TRAVIS_BUILD_DIR}/.travis/${TRAVIS_OS_NAME}.${TARGET_OS}.script.sh
-    - make -j4
-    - if [[ $TARGET_OS != win* ]]; then make tests && ./tests/tests; fi;
+    - . ${TRAVIS_BUILD_DIR}/.travis/script.sh
 after_script:
     - ccache -s
```

Comment:
If `ccache -s` output is irrelevant, then you should not run `ccache -s`. By ~10 lines, do you mean 2? No matter how many lines, it is worth because ShellCheck will check the script.